startup/routes.js

```diff
@@ -9,5 +9,5 @@ require("express-async-errors");
 module.exports = function (app) {
   app.use(express.json());
  
-
+  app.use('/api/room',rooms);
```

Comment:
In the route naming, not necessarily you need to add `/api`. However, make sure to discuss it with your teammates and decide on your preferred approach.